src/pkg/retention/dao/retention.go

```diff
@@ -85,9 +84,32 @@ func GetExecution(id int64) (*models.RetentionExecution, error) {
 	if err := o.Read(e); err != nil {
 		return nil, err
 	}
+	if err := fillStatus(e); err != nil {
+		return nil, err
+	}
 	return e, nil
 }
 
+// fillStatus the priority is InProgress Stopped Failed Succeed
+func fillStatus(exec *models.RetentionExecution) error {
```

Comment:
Need to fill the `StartTime` and `EndTime` either.

Reply:
Will remove EndTime, and calc on the fly
Keep StartTime